loader/src/ptracer/ptracer.cpp

```diff
@@ -16,6 +16,15 @@
 
 #include "utils.hpp"
 
+static void clearStackStringData(int pid, struct user_regs_struct& regs, const char *path) {
+    auto len = strlen(path) + 1;
```

Comment:
The goal of the project is to re-write it all to C99, so please avoid the use of C++ specific keywords.